modin/pandas/test/utils.py

```diff
@@ -645,6 +650,16 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
                 repr(fn(modin_df, **md_kwargs))
             if check_exception_type:
                 assert isinstance(md_e.value, type(pd_e))
+                if check_exception_type is not None and not isinstance(
+                    check_exception_type, bool
+                ):
+                    assert isinstance(
+                        md_e.value, check_exception_type
+                    ), "exception type not in the list of acceptable exception types"
+                if nonacceptable_exception_types is not None:
+                    assert not isinstance(
+                        md_e.value, tuple(nonacceptable_exception_types)
+                    ), "not acceptable exception type"
```

Comment:
Messages should contain the values of the variables from which the exception occurred. (to simplify debugging)

Reply:
Corrected.